src/sentry/apidocs/examples/replay_examples.py

```diff
@@ -74,24 +74,45 @@ class ReplayExamples:
             "Retrieve a collection of selectors for an organization.",
             value={
                 "data": [
-                    {
-                        "count_dead_clicks": 2,
-                        "count_rage_clicks": 1,
-                        "dom_element": "div#myid.class1.class2",
-                        "element": {
-                            "alt": "",
-                            "aria_label": "",
-                            "class": ["class1", "class2"],
-                            "id": "myid",
-                            "role": "",
-                            "tag": "div",
-                            "testid": "",
-                            "title": "",
+                    [
```

Comment:
Why is the response a doubly nested list?

Reply:
sorry I was trying to get it to pass CI, let me undo that